[Stream] Added docs for VOD clipping (#5400)
cloudflare/cloudflare-docs: content/stream/changelog.md

# Changelog

## 2022-08-02

> ### Clip video start and end times using the Stream API

Reviewer: Maybe simply saying 'Clip videos using the Stream API' would suffice since it mentions specifying the start and end times below and in the documentation?

Author: Good callout! Fixed

> You can now change the start and end times of a video uploaded to Cloudflare Stream. For more information, refer to [VOD clipping](/stream/edit-manage-videos/edit-videos/vod-clipping).

Author: Fixed!

> You can now change the start and end times of a video uploaded to Cloudflare Stream. For more information, refer to [VOD clipping](/stream/edit-manage-videos/edit-videos/video-clipping).

Reviewer: ```suggestion
You can now change the start and end times of a video uploaded to Cloudflare Stream. For more information, refer to [video clipping](/stream/edit-manage-videos/edit-videos/video-clipping).
```

> You can now change the start and end times of a video uploaded to Cloudflare Stream. For more information, refer to [video clipping](/stream/edit-manage-videos/edit-videos/video-clipping).

Reviewer 2: does this link work?

Author: It does...or should at least 😬. I double-checked it and added the slash at the end too because I realized I forgot it

> You can now change the start an end times of a video already uploaded to Cloudflare Stream. For more information, refer to [VOD clipping](/stream/edit-manage-videos/edit-videos/vod-clipping).

Reviewer: ```suggestion
You can now change the start and end times of a video uploaded to Cloudflare Stream. For more information, refer to [VOD clipping](/stream/edit-manage-videos/edit-videos/vod-clipping).
```
